### PR TITLE
fix: ACNA-1808 - move action generators to their own repos

### PR DESCRIPTION
**Pull request by shazron**
Related #204 

## How Has This Been Tested?

- npm test
- manual test
    - `aio plugins link` the app plugin
    - `npm link` this branch into the app plugin
    - `aio app init --standalone-app`
    - `aio app deploy`
    - `aio app test`
       
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: generators/index.js [hunk 1]
```diff
@@ -9,19 +9,12 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const excshell = require('@adobe/generator-app-excshell')
```
Comment (moritzraho) [inline generators/index.js:12]: That's about cleaning up some dead code right ?

**Review by moritzraho — COMMENTED** [review]
lgtm

File: generators/index.js (hunk 1 above)
Comment (shazron) [reply generators/index.js:12]: yes, everything goes through the Template Registry now for extensions

**Review by sandeep-paliwal — APPROVED** [review]
LGTM

**Pull request merged**